app/models/movie.rb

```diff
@@ -0,0 +1,6 @@
+class Movie < ApplicationRecord
+  has_many :rentals, dependent: :nullify
+  validates :title,  presence: true
+  validates :inventory, presence: true, numericality: {greater_than: 0}
```

Comment:
Running the seeds file, I'm getting validation errors, so your seeds file doesn't work with this validation.